syncd/Makefile.am

```diff
@@ -80,6 +80,10 @@ if SONIC_ASIC_PLATFORM_BROADCOM
 libSyncd_a_CXXFLAGS += -DMDIO_ACCESS_USE_NPU
 endif
 
+if SONIC_ASIC_PLATFORM_VS
```

Comment:
you are changind default behaviour by this, since by default vs switch is not considered as npu switch

Reply:
Because some code in MdioIpcServer.cpp is compiled with the condition MDIO_ACCESS_USE_NPU and seems not run in unit test, I tried to add the condition to the vs platform but it did not work. I will revert the change since it does not help the unit test.